tests/acceptance/topics-test.js

```diff
@@ -0,0 +1,86 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'atlantaember/tests/helpers/module-for-acceptance';
+
+const doesInclude = function (string, substrings) {
```

Comment:
Yea I wrote it.  I was thinking it belongs in the helpers maybe?  Maybe we can evolve like a `generalHelpers` object or something?